src/Console/Build/CompileScssCommand.php

```diff
@@ -135,7 +135,13 @@ protected function execute(InputInterface $input, OutputInterface $output)
                 ]
             );
 
-            $css = $licence_header . $css;
+            // Insert licence header after the "@charset" directive.
+            $css = preg_replace(
+                '/^(@charset .+;)/',
```

Comment:
Should not happen, but we should handle the fact that charset may not always be the single directive on its line.
```suggestion
                '/^(@charset [^;]+;)/',
```

Reply:
I'm not even sure that would be valid.